xcell/mappers/mapper_DESY1gc.py

```diff
@@ -64,6 +84,16 @@ def _get_mask(self):
         return mask
 
     def get_nz(self, dz=0):
+        """
+        Returns the mappers redshift \
+        distribtuion of sources from a file.
```

Comment:
sources? If any, these are lenses (in DES nomenclature). 
"Return the galaxy redshift distribution"